src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -712,6 +713,7 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
             <div className={displayInModal ? '' : 'be bdl-UnifiedShareForm'}>
                 <LoadingIndicatorWrapper isLoading={isFetching} hideContent>
                     {!hasExpandedSections && allShareRestrictionWarning}
+                    {upsellInlineNotice}
```

Comment:
Will the upsell inline notice look the same as the upgrade inline notice? They are both inline notices and thus should look exactly alike right?

Reply:
Also do we anticipate a need to wrap the passed in component in a div/container specific to this component in case a we want need to override styles, give it it's own testid etc? May not be an issue but just a thought.

Example:

 <div className="upsellWrapper">
 {upsellInlineNotice}
</div>

Reply:
@bfoxx1906 yes I added the upsellInlineNotice to use our new blueprint inline notice. We will eventually remove the older one in a follow up ticket.

I also agree with the change suggestion since I will be adding a margins